recipes/repeatscout/meta.yaml

```diff
@@ -7,17 +7,25 @@ source:
   sha256: bda6f782382f2b7dcb6a004b7da586d5046b3c12429b158e24787be62de6199c
 
 build:
-  number: 2
+  number: 3
   skip: True # [not linux]
 
 requirements:
   build:
     - {{ compiler('c') }}
   run:
+    - trf
+    - nseg
+    - perl
+    - perl-file-temp
 
 test:
   commands:
     - RepeatScout 2>&1 | grep Version
+    - build_lmer_table 2>&1 | grep Version
+    - filter-stage-1.prl -h 2>&1 | grep -i repeatscout
```

Comment:
Not that you have to change it now, but IMO using `-i` isn’t necessary because you know exactly what you search for. Just use the correct capitalization (as you did on the previous line).

Reply:
I thought it was safer